tests/drivers/spi/spi_loopback/boards/nrf51_pca10028.conf

```diff
@@ -1,10 +1,6 @@
 CONFIG_SPI_NRFX=y
 CONFIG_SPI_0=y
-CONFIG_SPI_0_IRQ_PRI=1
 CONFIG_SPI_0_NRF_SPI=y
-CONFIG_SPI_0_NRF_SCK_PIN=4
-CONFIG_SPI_0_NRF_MOSI_PIN=6
-CONFIG_SPI_0_NRF_MISO_PIN=5
```

Comment:
`CONFIG_SPI_LOOPBACK_CS_CTRL_GPIO_PIN` a few lines below (I can't add the comment there) should be updated, too. Pin 30 is now assigned to [MOSI](https://github.com/zephyrproject-rtos/zephyr/pull/9781/files#diff-dad5e9c5da79781871934105f03f7d92R57).

Reply:
Fixed, also removed collision in PCA10056.